govcd/nsxt_l2_vpn_tunnel.go

```diff
@@ -0,0 +1,281 @@
+package govcd
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+)
+
+// NsxtL2VpnTunnel extend your organization VDC by enabling virtual machines to
+// maintain their network connectivity across geographical boundaries while keeping
+// the same IP address. The connection is secured with a route-based IPSec tunnel between the two sides of the tunnel.
+// You can configure the L2 VPN service on an NSX-T edge gateway in your VMware Cloud Director environment
+// and create a L2 VPN tunnel. Virtual machines remain on the same subnet, which enables you to extend
+// your organization VDC by stretching its network. This way, an edge gateway at one site can provide
+// all services to virtual machines on the other site.
```

Comment:
Imho it looks better when not using 2nd person, but just a nitpick
```suggestion
// NsxtL2VpnTunnel extends an organization VDC by enabling virtual machines to
// maintain their network connectivity across geographical boundaries while keeping
// the same IP address. The connection is secured with a route-based IPSec tunnel between the two sides of the tunnel.
// The L2 VPN service can be configured on an NSX-T edge gateway in a VMware Cloud Director environment
// to create a L2 VPN tunnel. Virtual machines remain on the same subnet, which extends
// the organization VDC by stretching its network. This way, an edge gateway at one site can provide
// all services to virtual machines on the other site.
```

Reply:
Thanks for the suggestion, changed